lib/cavl/cavl.hpp

```diff
@@ -24,20 +24,28 @@
 
 #pragma once
 
+#include <array>
```

Comment:
All these changes in `cavl.hpp` are just copy/paste file replacement (from `pavel-kirienko@main`).